### PR TITLE
EXTERNAL_TOOLS

### PR DESCRIPTION
**Pull request by hascar**
Modified standalone tools menu depending from the value of EXTERNAL_TOOLS settings variable.

If EXTERNAL_TOOLS is True the menu tools show IPT, TaxtWEB, Taxonomy with external url.
If EXTERNAL_TOOLS is False the menu tools show Engine, IPT, TaxtWEB, Taxonomy with internal url.

Companion of https://github.com/gem/oq-engine/pull/9855

### CODE REVIEW

**Review by ptormene — APPROVED** [review]

File: openquakeplatform/settings.py [hunk 1]
```diff
@@ -14,6 +14,11 @@
 # Standalone flag to differentiate behaviors
 STANDALONE = True
 
+try:
+    EXTERNAL_TOOLS = True if os.environ['EXTERNAL_TOOLS'] == 'True' else False
```
Comment (nastasi-oq) [inline openquakeplatform/settings.py]: Please, use `os.getenv` instead of the 4 lines with try except statements

Comment (nastasi-oq) [reply openquakeplatform/settings.py]:
```suggestion
    EXTERNAL_TOOLS = True if os.getenv('EXTERNAL_TOOLS', "False") == "True" else False
```

File: openquakeplatform/settings.py [hunk 2]
```diff
@@ -14,6 +14,11 @@
 # Standalone flag to differentiate behaviors
 STANDALONE = True
 
+try:
```
Comment (nastasi-oq) [inline openquakeplatform/settings.py]:
```suggestion

```

File: openquakeplatform/settings.py [hunk 3]
```diff
@@ -14,6 +14,11 @@
...
+    EXTERNAL_TOOLS = True if os.environ['EXTERNAL_TOOLS'] == 'True' else False
+except KeyError:
```
Comment (nastasi-oq) [inline openquakeplatform/settings.py]:
```suggestion

```

File: openquakeplatform/settings.py [hunk 4]
```diff
@@ -14,6 +14,11 @@
...
+    EXTERNAL_TOOLS = False
```
Comment (nastasi-oq) [inline openquakeplatform/settings.py]:
```suggestion

```

File: openquakeplatform/settings.py (hunk 3 above)
Comment (hascar) [reply openquakeplatform/settings.py]: Fixed with this commit: https://github.com/gem/oq-platform-standalone/pull/60/commits/9ca123232d1e15d9464f903f132e59b66ec79f1f

File: openquakeplatform/settings.py (hunk 1 above)
Comment (hascar) [reply openquakeplatform/settings.py]: Fixed with this commit: https://github.com/gem/oq-platform-standalone/pull/60/commits/9ca123232d1e15d9464f903f132e59b66ec79f1f

**Review by nastasi-oq — APPROVED** [review]
LGTM

**Pull request merged**